modules/apps/portal-security/portal-security-script-management-web/src/main/resources/META-INF/resources/js/components/ScriptManagementContainer.tsx

```diff
@@ -0,0 +1,103 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayAlert from '@clayui/alert';
+import ClayButton from '@clayui/button';
+import {Text} from '@clayui/core';
+import {ClayCheckbox} from '@clayui/form';
+import {createResourceURL, fetch, openToast} from 'frontend-js-web';
+import React, {useState} from 'react';
+
+import './ScriptManagementContainer.scss';
+
+interface ScriptManagementContainerProps {
+	allowScriptContentBeExecutedOrIncluded: boolean;
+	baseResourceURL: string;
+}
+
+export default function ScriptManagementContainer({
+	allowScriptContentBeExecutedOrIncluded,
+	baseResourceURL,
+}: ScriptManagementContainerProps) {
+	const [allowScriptContent, setAllowScriptContent] = useState(
+		allowScriptContentBeExecutedOrIncluded
+	);
+
+	const handleSaveSystemConfiguration = async () => {
+		const response = await fetch(
+			createResourceURL(baseResourceURL, {
+				allowScriptContentBeExecutedOrIncluded: allowScriptContent,
+				p_p_resource_id:
+					'/system_settings/edit_script_management_configuration',
+			}).toString()
+		);
+
+		openToast({
+			message: response.ok
+				? Liferay.Language.get('your-request-completed-successfully')
+				: Liferay.Language.get('an-error-occurred'),
+			type: response.ok ? 'success' : 'danger',
+		});
+	};
+
+	return (
+		<div className="lfr__script-management-container">
+			<Text as="span" size={7} weight="bolder">
+				{Liferay.Language.get('script-management')}
+			</Text>
+
+			<ClayAlert displayType="info" title="Info:">
```

Comment:
Hey @danvb112, doesn't this title need to be localized?

Reply:
That's right, thanks!